crates/miden-tx/src/prover/mod.rs

```diff
@@ -118,6 +118,7 @@ impl LocalTransactionProver {
     pub fn prove(
         &self,
         tx_witness: TransactionWitness,
+        input_notes: InputNotes<InputNote>,
```

Comment:
It may make sense to put input notes into the `TransactionWitness` (the witness should be everything that one needs to prove a transaction).